library/bignum_mod_raw.c

```diff
@@ -127,7 +127,40 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+int mbedtls_mpi_mod_raw_to_mont_rep( mbedtls_mpi_uint *X,
+                                     const mbedtls_mpi_mod_modulus *m )
+{
+    const mbedtls_mpi_uint one = 1;
+    const size_t t_sz = ( m->limbs * 2 ) + 1;
```

Comment:
(a) for well-known precedences, having extra (unnecessary) parentheses slows down comprehension (b) `sz` isn't a useful suffix ("size" implies `size_t` implies bytes)

should change to something like

```suggestion
    const size_t t_limbs = m->limbs * 2 + 1;
```